### PR TITLE
Improve getting started guide

### PR DESCRIPTION
**Pull request by reflectronic**
- Adjust the instructions and examples to reflect more modern .NET
- Add a section about how to run the benchmarks (making sure you use Release)
- Fix the "Analyze" section, which was misleading, and add more useful information

### CODE REVIEW

File: docs/articles/guides/getting-started.md [hunk 1]
```diff
@@ -20,77 +18,76 @@ Write a class with methods that you want to measure and mark them with the `Benc
...
 
 ## Next steps
-
-BenchmarkDotNet provides a lot of features which help to high-quality performance research.
-If you want to know more about BenchmarkDotNet features, checkout the [Overview](../overview.md) page.
-If you want have any questions, checkout the [FAQ](../faq.md) page.
-If you didn't find answer for your question on this page, [ask it on gitter](https://gitter.im/dotnet/BenchmarkDotNet) or [create an issue](https://github.com/dotnet/BenchmarkDotNet/issues).
+BenchmarkDotNet provides features which aid high-quality performance research.
+If you want to know more about BenchmarkDotNet features, check out the [Overview](../overview.md) page.
+If you want have any questions, check out the [FAQ](../faq.md) page.
```
Comment (timcassell) [inline docs/articles/guides/getting-started.md:92]:
```diff
- If you want have any questions, check out the [FAQ](../faq.md) page.
+ If you have any questions, check out the [FAQ](../faq.md) page.
```

**Comment by reflectronic** [comment]
I wasn't planning on changing anything else, so I think it's good to merge as-is.

**Comment by AndreyAkinshin** [comment]
@reflectronic I think that it would be better to keep the old style with the Main method instead of the top-level statements that require C#10+: it will make the example more portable. With the Main method, the example can be copy-pasted without modifications to projects that are based on older C# versions.

**Comment by AndreyAkinshin** [comment]
@reflectronic is it OK if I rebase your branch, solve conflicts and revert the Main method by myself?

**Comment by reflectronic** [comment]
Sorry, I didn't see this until today... Yeah, it's alright if you rebase this and revert the changes to the benchmark code.

**Pull request merged**